server/service/handler.go

```diff
@@ -437,7 +437,7 @@ func attachFleetAPIRoutes(r *mux.Router, svc fleet.Service, config config.FleetC
 	ue.GET("/api/_version_/fleet/status/live_query", statusLiveQueryEndpoint, nil)
 
 	// Only Fleet MDM specific endpoints should be within the root /mdm/ path.
-	// please remember to update `service.mdmAppleConfigurationRequiredEndpoints`
+	// NOTE: remember to update `service.mdmAppleConfigurationRequiredEndpoints`
```

Comment:
Just to highlight this a bit more, as some text editors/IDE will make some comments (like NOTE and TODO) stand out.